api/v1/testkube.yaml

```diff
@@ -795,6 +987,8 @@ components:
         - success
         - error
 
+    
+
     ExecutionResult:
```

Comment:
perhaps we should rename this to ScriptExecutionResult - to avoid confusion with TestExecutionResult ?

Reply:
Will script/test be conifusing for testers? Aren't they used to test/suite naming ?

Reply:
but on the bottom there is execution result which is the same (just going from different direction) 
from test we need to go by 2 levels, from script we're getting it directly